Library/Formula/ucommon.rb

```diff
@@ -14,16 +12,29 @@ class Ucommon < Formula
   depends_on "pkg-config" => :build
   depends_on "gnutls"
 
+  needs :cxx11
+
+  # Upstream bug report:
+  #   http://lists.gnu.org/archive/html/bug-commoncpp/2015-01/msg00000.html
```

Comment:
Yeah, for me it means “the previous line continues here but I needed to break it to keep the text in ≤80 columns”.